osu.Game/Screens/Menu/IntroWelcome.cs

```diff
@@ -40,10 +46,15 @@ public IntroWelcome([CanBeNull] Func<MainMenu> createNextScreen = null)
         }
 
         [BackgroundDependencyLoader]
-        private void load(AudioManager audio)
+        private void load(AudioManager audio, IAPIProvider api)
         {
             if (MenuVoice.Value)
-                welcome = audio.Samples.Get(@"Intro/Welcome/welcome");
+            {
+                if (api.LocalUser.Value.IsSupporter)
```

Comment:
One thing to note is that this may not be ready in time for this check. But I think it's fine to ignore this for the time being. We'd want to address this for all usages rather than fix it only in this location. `Disclaimer` already has local support for handling the change of value, but again, this likely needs to be cached rather than checked on startup.